terraform/templates/modernisation-platform-environments-isolated/vpc.tf

```diff
@@ -1,7 +1,7 @@
 module "vpc" {
   #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
-  source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 5.0"
+
+  source  = "github.com/terraform-aws-modules/vpc/aws?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1
```

Comment:
this should be: `source  = "github.com/terraform-aws-modules/terraform-aws-vpc?ref=25322b6b6be69db6cca7f167d7b0e5327156a595" # v5.8.1`

Reply:
Missed this one. I will correct it